pkg/genericapiserver/server/options/server_run_options.go

```diff
@@ -66,7 +66,7 @@ type ServerRunOptions struct {
 func NewServerRunOptions() *ServerRunOptions {
 	return &ServerRunOptions{
 		AdmissionControl:            "AlwaysAdmit",
-		DefaultStorageMediaType:     "application/json",
```

Comment:
Thanks for pointing on it - I didn't think about it.

Do you think that setting it explicitly in kube-apiserver is a better approach?